src/function/gds/weakly_connected_components.cpp

```diff
@@ -15,83 +15,68 @@ using namespace kuzu::graph;
 namespace kuzu {
 namespace function {
 
-class ComponentIDs {
-public:
-    ComponentIDs(const table_id_map_t<offset_t>& maxOffsetMap, MemoryManager* mm) {
+struct WccComputationState {
```

Comment:
Ditto revert renaming and I would merge this with SCCIDs. Also why we have Wcc, WCC, Scc, SCC at this same time? Let's use one convention and if we decide to change we change in one PR all together.